RELEASE_NOTES.md

```diff
@@ -1,3 +1,11 @@
+#### 4.0.0
+
+* **Breaking:** Removed all occurrences of the argument `wrapDispatch` from the methods used to create a binding.  There is currently no migration path.  Please create an issue if this is a negative impact for you.
```

Comment:
For consistency, please have only one space between sentences.

Reply:
Done